inst/schema/ShapeResponseData.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Geo JSON object",
+  "description": "Schema for a Geo JSON object",
+  "type": "object",
+  "required": [ "type", "features" ],
+  "properties": {
+    "type": { "const": "FeatureCollection" },
+    "crs": { "type": "object" },
+    "name": { "tyep": "string" },
+    "features": {
+      "type": "array",
+      "items": { "type": "object" }
+    }
+  }
+}
```

Comment:
What's the best way to validate the GeoJSON? Just put this in as a placeholder. It would be cool if we could validate the GeoJSON against an online schema instead of creating a local copy.